### ISSUE TITLE
Add calendar view to help users quickly choose activities

### ISSUE THREAD

**Issue by pauljanes** [krico/jas#740]
Several of J's users have said it takes them too long to find the activities that they are looking to book. They have asked for a calendar type view so that they can quickly click on the classes they want on the calendar instead of having to scroll down the list.

If a calendar view is too much work then it would help to toggle the different activities so that for instance a Metafit user can quickly see the Metafit classes without having to see a mix of Metafit, Bootcamp morning and Bootcamp evening classes. Most of her users tend to either want to do Metafit or bootcamp morning or bootcamp evening. Not many of her users are looking for a mix of the 3 activity types.

So as I look at the activities page on the widget I would want to see

Metafit (activity type 1)
Bootcamp 07:00 (activity type 2)
Bootcamp 18:30 (activity type 3)

If I click on any of these items I would then want it to expand to see the individual activities of that type.

**Comment by wszarmach** [comment]
alternatively you could have week views, labels, or filters to allow the users to reduce the items to search through

**Comment by pauljanes** [comment]
Yep all ideas welcome if it helps user find what they want quicker

**Comment by pauljanes** [comment]
How about to start with we go for a quick and easy gain. We separate the list by activity type so they click on the activity type they want and then it expands and they see date sorted activities of the parent type. Similar to what we already did with the activity packages.
